File: 1448-maximum-69-number/1448-maximum-69-number.py
class Solution:
    def maximum69Number (self, num: int) -> int:
        n = num
        toAdd = 0
        i = 1
        while n > 0:
            if n % 10 == 6:
                toAdd = 3 * i
            n //= 10
            i *= 10
        return num + toAdd
        

File: 1448-maximum-69-number/1448-maximum-69-number.cpp
class Solution {
public:
    int maximum69Number (int num) {
        int n = num;
        int toAdd = 0;
        int i = 0;
        while(n > 0)
        {
            if(n % 10 == 6)
            {
                toAdd = 3 * pow(10, i);
            }
            n /= 10;
            i++;
        }
        return num + toAdd;
    }
};